changelog_unreleased/handlebars/pr-7306.md

```diff
@@ -0,0 +1,17 @@
+#### Support printing inline handlebars in html (through embed function) ([#7306](https://github.com/prettier/prettier/pull/7306) by [@dcyriller](https://github.com/dcyriller))
+
+<!-- prettier-ignore -->
+```
+<!-- Input -->
+<script type="text/x-handlebars-template">{{component 'hey'}}</script>
+
+<!-- Output (Prettier stable) -->
+<script type="text/x-handlebars-template">
+  {{component 'hey'}}
+</script>
+
+<!-- Output (Prettier master) -->
+<script type="text/x-handlebars-template">
+  {{component "hey"}}
```

Comment:
Doesn't really look like an improvement over Prettier stable. Please make the input uglier. :)

Reply:
Did you notice how it fixes a bug regarding quotes? ^^

I can make the change more obvious though.

Reply:
Will do (but I need a bit more time 😬)

Reply:
Here we go, it was worth it :)